handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -501,8 +524,52 @@ public OpenSslEngine remove(long ssl) {
         public void add(OpenSslEngine engine) {
             engines.put(engine.sslPointer(), engine);
         }
+
+        @Override
+        public OpenSslEngine get(long ssl) {
+            return engines.get(ssl);
+        }
     }
 
+    static void setKeyMaterial(long ctx, X509Certificate[] keyCertChain, PrivateKey key, String keyPassword)
+            throws SSLException {
+         /* Load the certificate file and private key. */
+        long keyBio = 0;
```

Comment:
This code is there forever and I not want to change it because of breakage now... I will revisit once we use our new tcnative with removed stuff and different package name